### PR TITLE
Add photo and quote generator.

### PR DESCRIPTION
**Pull request by officialcwang**

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -10,8 +10,12 @@
     <div id="content">
       <h1>My Portfolio</h1>
       <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
+      <p>Click here to get a cryptic quote from the Secret History:</p>
+      <button onclick="addRandomGreeting()">Hmm...</button>
+      <p></p>
```
Comment (luli0521) [inline portfolio/src/main/webapp/index.html:15]: Empty paragraph?

Comment (officialcwang) [reply portfolio/src/main/webapp/index.html:15]: Adds a space so that the image is not on the same line as the button.

Comment (luli0521) [reply portfolio/src/main/webapp/index.html:15]: An option is to use flexbox: https://www.w3schools.com/css/tryit.asp?filename=trycss3_flexbox_flex-direction_column

Comment (officialcwang) [reply portfolio/src/main/webapp/index.html:15]: Okay, thanks!

**Pull request merged**